src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -33,7 +34,21 @@ case class DatabricksContractPrices(
                                      jobsLightDBUCostUSD: Double = 0.10
                                    )
 
-case class ApiEnv(isLocal: Boolean, workspaceURL: String, rawToken: String, packageVersion: String)
+case class ApiEnv(isLocal: Boolean, workspaceURL: String, rawToken: String, packageVersion: String,successBatchSize:Int=0,errorBatchSize:Int=0,runID:String="")
```

Comment:
A 3rd set of defaults for batch sizes -- these should all be defaulted to the same, reasonable value.

Curious as to if we can set the package version as a function inside the case class so that it can be automatic. So remove `packageVersion` from the signature completely and create a function called `packageVersion` that get's the package version in the same way as in the config.scala

Reply:
We don't have any use of packageVersion inside API call, i guess we can remove it from APIEnv case class.

Reply:
why are you defaulting batch sizes to 0? I strongly suggest we default this to something reasonable. 0 is prob the worst default as it will throw strange errors